code/modules/vore/eating/living_vr.dm

```diff
@@ -587,3 +587,87 @@
 	var/new_color = input(src,"Select a new color","Body Glow",glow_color) as color
 	if(new_color)
 		glow_color = new_color
+
+/mob/living/proc/eat_trash()
+	set name = "Eat Trash"
+	set category = "Abilities"
+	set desc = "Consume held garbage."
+
+	var/obj/item/I = get_active_hand()
+	if(!I)
+		to_chat(src, "<span class='notice'>You are not holding anything.</span>")
+		return
+	var/list/edible_trash = list(/obj/item/trash,
```

Comment:
Should be in `global_lists_vr.dm` probably. Otherwise this list is created and then garbage collected (no pun intended) each time this proc is run.